### PR TITLE
[Consignments] Wire up Recently Sold

### PR DESCRIPTION
**Pull request by damassi**
Depends on https://github.com/artsy/metaphysics/pull/2418 (all of those fragment diffs should go away once MP pr is merged) 

This finishes up the new consign home (minus tests) and does a few things:
- Cleans up component structure by organizing things into files and other misc
- Adds Recently Sold bar back and wires up to real data

Todo in next PR:
- Write tests

recently-sold

#trivial 
#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Missing Test Files:

- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/ArtistList-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/Footer-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/Header-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/HowItWorks-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/RecentlySold-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/TextContainer-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx [hunk 1]
```diff
@@ -33,17 +31,15 @@ export const ConsignmentsHome: React.FC<Props> = ({ artists, isLoading }) => {
   }
 
   return (
-    <ProvidePlaceholderContext>
```
Comment (damassi) [inline src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx]: This is how I built out the placeholder outside of a real loading context

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Missing Test Files:

- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/ArtistList-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/Footer-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/Header-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/HowItWorks-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/RecentlySold-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/TextContainer-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Missing Test Files:

- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/ArtistList-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/Footer-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/Header-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/HowItWorks-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/RecentlySold-tests.tsx`
- `src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/__tests__/TextContainer-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/__generated__/getGeminiCredentialsForEnvironmentMutation.graphql.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 /* tslint:disable */
 /* eslint-disable */
-/* @relayHash fa82c715ffc5d780dcee281eb4d944f5 */
+/* @relayHash 2e13fda169954c5cc132e5e3a1dd1db8 */
 
 import { ConcreteRequest } from "relay-runtime";
 export type RequestCredentialsForAssetUploadInput = {
+    name: string;
```
Comment (pepopowitz) [inline src/__generated__/getGeminiCredentialsForEnvironmentMutation.graphql.ts:7]: At first I assumed this relay churn was all prettier or something, but it looks like it's mostly reordering fields into a non-alphabetical order. Do you know what all this is about @damassi ?

File: src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx [hunk 3]
```diff
@@ -1,285 +1,52 @@
...
-      <Separator my={3} />
-
-      <ArtistsModule artists={artists} />
-
-      <Separator my={3} />
-
-      <FooterCTA handleCTAPress={handleCTAPress} />
+      <Join separator={<Separator my={3} />}>
+        <Header onCTAPress={presentSubmissionModal} />
+        <RecentlySold artists={artists} isLoading={isLoading} />
```
Comment (pepopowitz) [inline src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx:37]: I am not going to ask you to change this, but fwiw I'm not a fan of `isLoading` props. In general they add "modes" to components that make them more complex; in this specific example, I preferred the old way of building a placeholder component that was distinct from the real component, but composed itself using many of the same building blocks.

File: src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx [hunk 4]
```diff
@@ -299,8 +66,33 @@ export const ConsignmentsHomeQueryRenderer: React.FC = () => {
...
+  "4e975df46ba7120001001fe2": "Mr. Brainwash",
+  "4f5f64c13b555230ac000004": "Nina Chanel Abney",
+  "4d8b92734eb68a1b2c00010c": "Roy Lichtenstein",
+  "4d9b330cff9a375c2f0031a8": "Sterling Ruby",
+  "551bcaa77261692b6f181400": "Stik",
+  "4d8b92bb4eb68a1b2c000452": "Takashi Murakami",
+  "4ef3c0ee9f1ce1000100022f": "Tomoo Gokita",
+})
+
+// console.log(FIXME_MICROFUNNEL_ARTISTS)
```
Comment (pepopowitz) [inline src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx:98]:
```suggestion
```

File: src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx (hunk 3 above)
Comment (damassi) [reply src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/index.tsx:37]: My thinking is: the cognative benefits of having a single boolean that toggles a state inherent to Eigens nature (loading data) weighs out. Stuff doesn't need to be duplicated, can be co-located, and follows a very explict control flow -- the number of modes are exactly two: loading or not loading.

File: src/__generated__/getGeminiCredentialsForEnvironmentMutation.graphql.ts (hunk 2 above)
Comment (damassi) [reply src/__generated__/getGeminiCredentialsForEnvironmentMutation.graphql.ts:7]: I think this just needs to be rebased

**Comment by damassi** [comment]
#mergeOnGreen

**Pull request merged**